README.md

```diff
@@ -302,14 +302,15 @@ _Links to various users of Home Assistant that regularly publish Home Assistant
 
 _Sit back, relax, watch, and learn._
 
-* [BRUH](https://www.youtube.com/channel/UCLecVrux63S6aYiErxdiy4w) - Ben has great tutorials for getting started, unfortunately, inactive lately.
+* [BRUH](https://www.youtube.com/channel/UCLecVrux63S6aYiErxdiy4w) - Ben has great tutorials for getting started, was inactive for a peroid but is back making videos again.
```

Comment:
~for a peroid but is back making videos again~
"for a period, but is back making videos again."